src/hotspot/share/runtime/vmStructs.cpp

```diff
@@ -941,8 +941,6 @@ typedef HashtableEntry<InstanceKlass*, mtClass>  KlassHashtableEntry;
   c2_nonstatic_field(JVMState,                 _method,                                       ciMethod*)                             \
   c2_nonstatic_field(JVMState,                 _map,                                          SafePointNode*)                        \
                                                                                                                                      \
-  c2_nonstatic_field(SafePointNode,            _jvms,                                         JVMState* const)                       \
-                                                                                                                                     \
```

Comment:
Don't remove it. It is referenced from HotSpot Servicability agent:
https://github.com/openjdk/jdk/blob/master/src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/opto/SafePointNode.java#L45

Reply:
Thank you for pointing out this.  I wasn't aware of this agent thing. 
It turns out I can neither move SafePointNode::_jvms to the protected zone nor take off its const qualifier. 
so I revert my changes except removing "virtual" for the SafePointNode::jvms().

Reply:
I reverted files which need to update copyright years.